src/components/candidatesListItem.js

```diff
@@ -0,0 +1,33 @@
+import React from "react"
+import Bar from "./bar"
+import styles from "./candidatesListItem.module.scss"
+
+const percentFormatter = Intl.NumberFormat("en-US", { style: "percent" })
+
+function formatPercent(value) {
+  return percentFormatter.format(value)
+}
+
+export default ({ Name }) => {
+  const percent = formatPercent(83455 / 123456)
```

Comment:
do we need to do this math every time? why the magic numbers?

Reply:
That's there until we get real numbers hooked up; right now the mock API doesn't provide that information so this was there so I could style the element properly.